### ISSUE TITLE
Warning  FailedScheduling 0/6 nodes are available: 6 Insufficient topolvm.cybozu.com/capacity.

### ISSUE THREAD

**Issue by pravinrajput** [topolvm/topolvm#235]
**Describe the bug**
Deployed topolvm as service & deployment on amazon EKS cluster, and then deployed Nginx to use `topolvm-provisioner` storage class. But nginx pod is not coming up, it is in a pending state. Getting following `Insufficient` error. 

```
sudoers:topolvm pravinsinghrajput$ kubectl get sc | grep topolvm-provisioner
topolvm-provisioner   topolvm.cybozu.com             Delete          WaitForFirstConsumer   true                   117m

sudoers:topolvm pravinsinghrajput$ kubectl get pvc
NAME          STATUS    VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS          AGE
topolvm-pvc   Pending                                      topolvm-provisioner   99m

sudoers:topolvm pravinsinghrajput$ kubectl get po
NAME      READY   STATUS    RESTARTS   AGE
nginx-1   0/1     Pending   0          70m

sudoers:topolvm pravinsinghrajput$ kubectl describe po nginx-1
Name:         nginx-1
Namespace:    default
Priority:     0
Node:         <none>
Labels:       app.kubernetes.io/name=nginx-1
Annotations:  capacity.topolvm.cybozu.com/00default: 1073741824
              kubernetes.io/psp: eks.privileged
Status:       Pending
IP:           
IPs:          <none>
Containers:
  nginx:
    Image:      nginx
    Port:       <none>
    Host Port:  <none>
    Limits:
      topolvm.cybozu.com/capacity:  1
    Requests:
      topolvm.cybozu.com/capacity:  1
    Environment:                    <none>
    Mounts:
      /var/run/secrets/kubernetes.io/serviceaccount from default-token-t5ltl (ro)
      /var/www/html from vol (rw)
Conditions:
  Type           Status
  PodScheduled   False 
Volumes:
  vol:
    Type:       PersistentVolumeClaim (a reference to a PersistentVolumeClaim in the same namespace)
    ClaimName:  topolvm-pvc
    ReadOnly:   false
  default-token-t5ltl:
    Type:        Secret (a volume populated by a Secret)
    SecretName:  default-token-t5ltl
    Optional:    false
QoS Class:       BestEffort
Node-Selectors:  kubernetes.io/hostname=ip-10-141-10-238.ec2.internal
Tolerations:     node.kubernetes.io/not-ready:NoExecute for 300s
                 node.kubernetes.io/unreachable:NoExecute for 300s
Events:
  Type     Reason             Age                    From                Message
  ----     ------             ----                   ----                -------
  Normal   NotTriggerScaleUp  9m51s (x48 over 67m)   cluster-autoscaler  pod didn't trigger scale-up (it wouldn't fit if a new node is added): 2 max node group size reached, 4 Insufficient topolvm.cybozu.com/capacity
  Normal   NotTriggerScaleUp  4m48s (x339 over 70m)  cluster-autoscaler  pod didn't trigger scale-up (it wouldn't fit if a new node is added): 4 Insufficient topolvm.cybozu.com/capacity, 2 max node group size reached
  Warning  FailedScheduling   4m26s (x47 over 70m)   default-scheduler   0/6 nodes are available: 6 Insufficient topolvm.cybozu.com/capacity.
```

**Environments**
- Version: AWS EKS K8s Version: v1.17
- OS: Amazon Linux AMI 2

**To Reproduce**
1. Installed & started lvmd on worker node (i3.large) instance type 
```shell
[root@ip-10-141-10-238 topolvm]# ls
lvmd.sock
[root@ip-10-141-10-238 topolvm]# service lvmd status
Redirecting to /bin/systemctl status lvmd.service
● lvmd.service - lvmd for TopoLVM
   Loaded: loaded (/usr/lib/systemd/system/lvmd.service; disabled; vendor preset: disabled)
   Active: active (running) since Mon 2020-12-28 11:21:04 UTC; 3h 31min ago
  Process: 19015 ExecStartPre=/bin/mkdir -p /run/topolvm (code=exited, status=0/SUCCESS)
 Main PID: 19033 (lvmd)
    Tasks: 10
   Memory: 3.8M
   CGroup: /system.slice/lvmd.service
           └─19033 /bin/lvmd

Dec 28 11:21:04 ip-10-141-10-238.ec2.internal systemd[1]: Starting lvmd for TopoLVM...
Dec 28 11:21:04 ip-10-141-10-238.ec2.internal systemd[1]: Started lvmd for TopoLVM.
Dec 28 11:21:04 ip-10-141-10-238.ec2.internal lvmd[19033]: 2020-12-28T11:21:04.928882Z ip-10-141-10-238.ec2.internal lvmd info: "configuration file loaded: " device_classes...lvmd.sock"
Hint: Some lines were ellipsized, use -l to show in full.
```
2. Created namespace [topolvm-system](https://github.com/topolvm/topolvm/blob/master/deploy/manifests/base/namespace.yaml)
3. Created certificate using cert-manager, used [certificates.yaml](https://github.com/topolvm/topolvm/blob/master/deploy/manifests/base/certificates.yaml)
4. Deployed [topolvm-scheduler](https://github.com/topolvm/topolvm/blob/master/deploy/manifests/overlays/deployment-scheduler/scheduler.yaml) This will create a classic load balancer in AWS, once it is ready modified `urlPrefix` in [scheduler-policy.cfg](https://github.com/topolvm/topolvm/blob/master/deploy/scheduler-config/scheduler-policy.cfg)
5. Executed the following command, to apply the remaining changes of topolvm i.e. controller, scheduler, nodes, rbac, polices etc.
```shell 
kustomize build ./deploy/manifests/overlays/deployment-scheduler | kubectl apply -f -
```
6. Created storage class [topolvm-provisioner](https://github.com/topolvm/topolvm/blob/master/deploy/manifests/base/provisioner.yaml)
7. Deployed nginx app to use topolvm-provisioner 
nginx.yaml

```yaml
kind: PersistentVolumeClaim
apiVersion: v1
metadata:
  name: topolvm-pvc
spec:
  accessModes:
  - ReadWriteOnce
  resources:
    requests:
      storage: 1Gi
  storageClassName: topolvm-provisioner 
---
apiVersion: v1
kind: Pod
metadata:
  name: nginx-1
  labels:
    app.kubernetes.io/name: nginx-1
spec:
  containers:
  - name: nginx
    image: nginx
    volumeMounts:
    - mountPath: /var/www/html
      name: vol
  nodeSelector:
      kubernetes.io/hostname: ip-10-141-10-238.ec2.internal
  volumes:
    - name: vol
      persistentVolumeClaim:
        claimName: topolvm-pvc 
```

8. topolvm services, pods & deployments are running & up

```shell
sudoers:topolvm pravinsinghrajput$ kubectl get po -n topolvm-system
NAME                                READY   STATUS    RESTARTS   AGE
controller-675776c48b-7s78t         5/5     Running   12         112m
node-dpbzv                          3/3     Running   0          112m
topolvm-scheduler-bffdf8f7b-djs9v   1/1     Running   0          112m

sudoers:topolvm pravinsinghrajput$ kubectl get svc -n topolvm-system
NAME                TYPE           CLUSTER-IP       EXTERNAL-IP                                                               PORT(S)          AGE
controller          ClusterIP      X.X.X.X   <none>                                                                    443/TCP          112m
topolvm-scheduler   LoadBalancer   X.X.X.X    a0c00a88a3d3e4d1ab1891c9202de744-XXXXXXXXX.us-east-1.elb.amazonaws.com   9251:32272/TCP   128m

sudoers:topolvm pravinsinghrajput$ kubectl get deploy -n topolvm-system
NAME                READY   UP-TO-DATE   AVAILABLE   AGE
controller          1/1     1            1           113m
topolvm-scheduler   1/1     1            1           129m
```

**Expected behavior**
nginx pod starts with `topolvm-provisioner` storage type. 

**Additional context**
Logs for reference:

`topolvm-controller` 

```shell
sudoers:topolvm pravinsinghrajput$ kubectl -n topolvm-system logs controller-675776c48b-7s78t topolvm-controller --tail=50
{"level":"info","ts":1609165524.4236534,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609165584.4231,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609165644.4231822,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609165704.422965,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609165764.4231207,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609165824.4231234,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609165884.422845,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609165944.4239929,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166004.4229457,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166064.4232333,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166124.4231813,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166184.4232855,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166244.423123,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166304.4228964,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166364.4238236,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166424.4231658,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166484.42308,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166544.4232152,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166604.4228005,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166664.4228444,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166724.422956,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166784.4229684,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166844.423243,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166904.423,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166964.4230201,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167024.42312,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167084.4229324,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167144.4228845,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167204.4227893,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167264.422957,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167324.4231353,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167384.4228415,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167444.4230075,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167504.4231112,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167564.4229972,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167624.4231312,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167684.4230409,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167744.4231262,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167804.422983,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167864.4229167,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167924.4231164,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167984.4226613,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168044.423185,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168104.4232743,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168164.422965,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168224.4230642,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168284.4228835,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168344.4231975,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168404.4242418,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168464.4230835,"logger":"driver.identity","msg":"Probe","req":""}
```

`topolvm-scheduler`

```shell
sudoers:topolvm pravinsinghrajput$ kubectl -n topolvm-system logs topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler --tail=50
2020-12-28T15:07:19.988317Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="c7cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000016861 type="access" url="/status"
2020-12-28T15:07:29.988335Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="f8cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000016003 type="access" url="/status"
2020-12-28T15:07:39.988424Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="f9cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.00001282 type="access" url="/status"
2020-12-28T15:07:49.988313Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="facbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000013558 type="access" url="/status"
2020-12-28T15:07:59.988487Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="fbcbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000014392 type="access" url="/status"
2020-12-28T15:08:09.988512Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="fccbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000013794 type="access" url="/status"
2020-12-28T15:08:19.988328Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="fdcbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000014018 type="access" url="/status"
2020-12-28T15:08:29.988307Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="fecbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000017257 type="access" url="/status"
2020-12-28T15:08:39.988339Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="ffcbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000015299 type="access" url="/status"
2020-12-28T15:08:49.988325Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="f0cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000013112 type="access" url="/status"
2020-12-28T15:08:59.988280Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="f1cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.0000149 type="access" url="/status"
2020-12-28T15:09:09.988370Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="f2cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000022015 type="access" url="/status"
2020-12-28T15:09:19.988470Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="f3cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000016732 type="access" url="/status"
2020-12-28T15:09:29.988448Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="f4cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000017282 type="access" url="/status"
2020-12-28T15:09:39.988472Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="f5cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000013978 type="access" url="/status"
2020-12-28T15:09:49.988208Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="f6cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000009313 type="access" url="/status"
2020-12-28T15:09:59.988404Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="f7cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000009802 type="access" url="/status"
2020-12-28T15:10:09.988358Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="e8cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000037043 type="access" url="/status"
2020-12-28T15:10:19.988307Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="e9cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000011406 type="access" url="/status"
2020-12-28T15:10:29.988373Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="eacbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000015677 type="access" url="/status"
2020-12-28T15:10:39.988251Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="ebcbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000013446 type="access" url="/status"
2020-12-28T15:10:49.988299Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="eccbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000010501 type="access" url="/status"
2020-12-28T15:10:59.988346Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="edcbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000014662 type="access" url="/status"
2020-12-28T15:11:09.988344Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="eecbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000015481 type="access" url="/status"
2020-12-28T15:11:19.988397Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="efcbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000016518 type="access" url="/status"
2020-12-28T15:11:29.988294Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="e0cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000015024 type="access" url="/status"
2020-12-28T15:11:39.988342Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="e1cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000016467 type="access" url="/status"
2020-12-28T15:11:49.988355Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="e2cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000012321 type="access" url="/status"
2020-12-28T15:11:59.988374Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="e3cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.00004068 type="access" url="/status"
2020-12-28T15:12:09.988281Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="e4cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000013137 type="access" url="/status"
2020-12-28T15:12:19.988323Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="e5cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000015152 type="access" url="/status"
2020-12-28T15:12:29.988312Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="e6cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000014585 type="access" url="/status"
2020-12-28T15:12:39.988582Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="e7cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000015393 type="access" url="/status"
2020-12-28T15:12:49.988288Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="18cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000014649 type="access" url="/status"
2020-12-28T15:12:59.988282Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="19cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000014169 type="access" url="/status"
2020-12-28T15:13:09.988271Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="1acbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000012622 type="access" url="/status"
2020-12-28T15:13:19.988491Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="1bcbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000015186 type="access" url="/status"
2020-12-28T15:13:29.988280Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="1ccbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000011337 type="access" url="/status"
2020-12-28T15:13:39.988320Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="1dcbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000016047 type="access" url="/status"
2020-12-28T15:13:49.988298Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="1ecbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000013046 type="access" url="/status"
2020-12-28T15:13:59.988299Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="1fcbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000043134 type="access" url="/status"
2020-12-28T15:14:09.988301Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="10cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000015889 type="access" url="/status"
2020-12-28T15:14:19.989466Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="11cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000009589 type="access" url="/status"
2020-12-28T15:14:29.988412Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="12cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000017589 type="access" url="/status"
2020-12-28T15:14:39.988424Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="13cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000014224 type="access" url="/status"
2020-12-28T15:14:49.988359Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="14cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000013835 type="access" url="/status"
2020-12-28T15:14:59.988845Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="15cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000015275 type="access" url="/status"
2020-12-28T15:15:09.988296Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="16cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000012598 type="access" url="/status"
2020-12-28T15:15:19.988265Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="17cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000015443 type="access" url="/status"
2020-12-28T15:15:29.988337Z topolvm-scheduler-bffdf8f7b-djs9v topolvm-scheduler info: "well: access" http_host="10.141.11.116:9251" http_method="GET" http_status_code=200 http_user_agent="kube-probe/1.17+" protocol="HTTP/1.1" remote_ipaddr="10.141.10.238" request_id="08cbe1c6-e153-492a-1f10-415c3235eaa9" request_size=0 response_size=2 response_time=0.000015294 type="access" url="/status"
```

`topolvm-node`

```shell
sudoers:topolvm pravinsinghrajput$ kubectl logs node-dpbzv -n topolvm-system -c topolvm-node --tail=50
{"level":"info","ts":1609165671.4545228,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609165731.4538546,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609165791.4536526,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609165851.4538908,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609165911.4536915,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609165971.4536586,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166031.4547877,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166091.4538066,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166151.453634,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166211.4537668,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166271.4537792,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166331.4537807,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166391.4537003,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166451.4536765,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166511.4535785,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166571.4539907,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166631.4538581,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166691.4538805,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166751.4534311,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166811.4538531,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166871.4535065,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166931.4537702,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609166991.4536085,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167051.4536633,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167111.453665,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167171.4538794,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167231.4535851,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167291.4537127,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167351.4536195,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167411.4536066,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167471.4545362,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167531.4536679,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167591.4536877,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167651.453784,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167711.453699,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167771.4537973,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167831.4536917,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167891.4536166,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609167951.4536862,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168011.4539404,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168071.4536798,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168131.453778,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168191.454621,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168251.4535825,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168311.453661,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168371.4536827,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168431.45345,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168491.45389,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168551.4535828,"logger":"driver.identity","msg":"Probe","req":""}
{"level":"info","ts":1609168611.4538002,"logger":"driver.identity","msg":"Probe","req":""}
```

**Comment by toshipp** [comment]
@pravinrajput 
It seems volume groups on nodes do not have enough capacity.
Would you confirm the free space of VGs by `vgs` command?

**Comment by ymmt2005** [comment]
@toshipp @pravinrajput 
This is a typical symptom when topolvm-scheduler is not properly configured for kube-scheduler.

**Comment by pravinrajput** [comment]
@ymmt2005 
Could you please help me with the steps to configure topolvm-scheduler with kube-scheduler?
Thanks!

**Comment by llamerada-jp** [comment]
Hi @pravinrajput ,

From the logs of topolvm-scheduler in your environment, it looks like kube-scheduler is not accessing topolvm-scheduler. If topolvm-scheduler is accessed, it should output a log like the following.

```
2021-01-05T02:14:28.309112Z topolvm-e2e-control-plane topolvm-scheduler info: "well: access" http_host="127.0.0.1:9251" http_method="POST" http_status_code=200 http_user_agent="Go-http-client/1.1" protocol="HTTP/1.1" remote_ipaddr="127.0.0.1" request_id="6499d1c1-6a3d-2c01-187e-37c74b4b5b39" request_size=17908 response_size=14505 response_time=0.000981101 type="access" url="/predicate"
2021-01-05T02:14:28.310943Z topolvm-e2e-control-plane topolvm-scheduler info: "well: access" http_host="127.0.0.1:9251" http_method="POST" http_status_code=200 http_user_agent="Go-http-client/1.1" protocol="HTTP/1.1" remote_ipaddr="127.0.0.1" request_id="6399d1c1-6a3d-2c01-187e-37c74b4b5b39" request_size=17908 response_size=83 response_time=0.000485 type="access" url="/prioritize"
```

Have you configured any of the following settings?
https://github.com/topolvm/topolvm/tree/master/deploy#configure-kube-scheduler

Could you please confirm that you can manually access topolvm-scheduler from the kube-scheduler node? Also, could you please check if you are using cluster-ip instead of external-ip in the configuration of kube-scheduler?

**Comment by pravinrajput** [comment]
@llamerada-jp 
I am using AWS EKS managed k8 platform. I don't have access to kube-scheudler.

**Comment by toshipp** [comment]
@pravinrajput 
Currently, topolvm needs scheduler-extender, so it does not work correctly on EKS.
As of k8s 1.19, thanks to the Storage Capacity Tracking feature, scheduler-extender is no longer needed.
The issue is already filed as #193.

If you want to use topolvm on k8s 1.18 or below, you should deploy k8s manually.

**Comment by pravinrajput** [comment]
@toshipp Thanks for the update. I will wait for this new feature.

**Comment by arianvp** [comment]
Given the feature is implemented now; all there is to do is wait for EKS to release 1.21 :)

**Comment by toshipp** [comment]
@imul-p 
I don't know about EKS, so I can't give you an exact answer. Theoretically, if we deploy the second scheduler that is configured to use topolvm-scheduler and set schedulerName of pods to the second scheduler, it would work.
BTW, If you don't need prioritization by VG capacity, topolvm-scheduler is not required and our helm chart is now shipped without enabling topolvm-scheduler.
